app/views/games/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @games.each do |game| %>
```

Comment:
we are now listing 7000 games.... perhaps we should just list the first 20 or so?

Reply:
Yeah, good point